rust/chains/tw_pactus/src/transaction.rs

```diff
@@ -0,0 +1,410 @@
+use std::fmt::Debug;
+use std::str::FromStr;
+
+use tw_coin_entry::error::prelude::{SigningError, SigningErrorType, SigningResult};
+use tw_hash::blake2::blake2_b;
+use tw_keypair::ed25519::sha512::{PrivateKey, PublicKey};
+use tw_keypair::ed25519::Signature;
+use tw_keypair::traits::SigningKeyTrait;
+use tw_proto::Pactus;
+
+use crate::address::Address;
+use crate::amount::Amount;
+use crate::encoder::error::Error as EncoderError;
+use crate::encoder::{deserialize, Decodable, Encodable};
+
+const VERSION_LATEST: u8 = 1;
+const FLAG_NOT_SIGNED: u8 = 0x02;
+
+#[derive(Debug, Clone)]
```

Comment:
How about adding `Copy` derive and `repr(u8)` here?

Reply:
It's done, thanks.